applications/openshift/api-server/audit_log_forwarding_is_enabled/rule.yml

```
@@ -0,0 +1,32 @@
+prodtype: ocp4
+
+title: Ensure that the audit logs are forwarded off the cluster for retention
+
+description: |-
+  OpenShift audit works at the API server level, logging all requests coming to the server.
+  Audit is on by default and the best practice is to ship audit logs off the cluster for retention. 
+
+rationale: |-
+  Retaining logs ensures that you can go back in time and investigate or correlate any events.
+  Doing so off cluster ensures that an attacker that has access to the cluster won't be able to
+  tamper with the logs, as they'll be stored off-site.
```

```suggestion
  Retaining logs ensures the ability to go back in time to investigate or correlate any events.
  Offloading audit logs from the cluster ensures that an attacker that has access to the cluster will not be able to
  tamper with the logs because of the logs being stored off-site.
```

It's best not to use contractions or "you" in these rationale as it makes them more official.

---

Let's write guidelines for writing content.